File: examples/hanoi.py
import datetime, time
import sys

def hanoi(n):
    if n == 1:
        return 1
    else:
        return hanoi(n-1) + hanoi(n-1) + 1    

t = datetime.datetime.now()
res = hanoi(25)
print("OK: " + str(res) if res == 33554431 else "WRONG: " + str(res))
print("time:", (datetime.datetime.now()-t))

File: examples/hanoi.js
eval(`function hanoi(n){
    if(n == 1){
        return 1
    } else {
        return hanoi(n-1) + hanoi(n-1) + 1
    }
}`)

let t = Date.now()
let res = hanoi(25);
console.log(res == 33554431 ?  "OK: " + res : "WRONG: " + res, "time:", Date.now()-t)